lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -145,9 +145,10 @@ def answer_label
       def answer_labels
         Array.wrap(@current["value"]).map do |answer_idx|
           begin
-            answer_string = workflow_at_version.tasks[@current['task']]['answers'][answer_idx]['label']
+            task_answer = workflow_at_version.tasks[@current['task']]['answers'][answer_idx]
```

Comment:
Could we use `Hash#dig` instead here to fetch nested values (works across hashes with arrays with hashes just fine)? That way we don't have to capture all `NoMethodError`s. But we'd get a `nil` back, and we wouldn't be able to distinguish between a label that is configured to be `nil` and one that is not configured. Don't think that's an issue, but sanity check me here.